lib/routes/index.js

```diff
@@ -385,6 +385,12 @@ app.route('/voies/:voieId/batch')
     res.send(batchUpdate)
   }))
 
+app.route('/voies/:voieId/convertToToponyme')
```

Comment:
J'utiliserais plutôt le kebab case pour convertToToponyme => convert-to-toponyme. Pour rester sur le même style que les autres routes (/bases-locales/:baseLocaleId/habilitation/email/send-pin-code, /bases-locales/:baseLocaleId/habilitation/email/validate-pin-code)